api/client/network.go

```diff
@@ -224,14 +225,13 @@ func (cli *DockerCli) CmdNetworkInspect(args ...string) error {
 	var networks []types.NetworkResource
 	buf := new(bytes.Buffer)
 	for _, name := range cmd.Args() {
-		obj, _, err := readBody(cli.call("GET", "/networks/"+name, nil, nil))
+		obj, statusCode, err := readBody(cli.call("GET", "/networks/"+name, nil, nil))
 		if err != nil {
-			if strings.Contains(err.Error(), "not found") {
-				fmt.Fprintf(cli.err, "Error: No such network: %s\n", name)
-			} else {
+			if statusCode != http.StatusNotFound {
 				fmt.Fprintf(cli.err, "%s\n", err)
 			}
 			status = 1
+			fmt.Fprintf(cli.err, "Error: No such network: %s\n", name)
```

Comment:
Shouldn't this also be a if/else, same as in https://github.com/docker/docker/pull/18463?

If you change it, please also swap the if/else, and use `statusCode == http.StatusNotFound`, like I commented in https://github.com/docker/docker/pull/18463/files#r46802028